yb-voyager/cmd/analyzeSchema.go

```diff
@@ -61,6 +61,7 @@ var (
 	optionalCommaSeperatedTokens = `[^,]+(?:,[^,]+){0,}`
 	commaSeperatedTokens         = `[^,]+(?:,[^,]+){1,}`
 	unqualifiedIdent             = `[a-zA-Z0-9_]+`
+	clause                       = `[A-Z]+`
```

Comment:
How about calling this variable as `fetchLocation`?
Should it also include lower case characters e.g. `fetch first from`?